### PR TITLE
tests: integration test  for system reload

### PR DESCRIPTION
**Pull request by mvo5**
Build on top of https://github.com/snapcore/snapd/pull/2697

### CODE REVIEW

**Comment by niemeyer** [comment]
Thanks!

File: tests/main/snap-service/task.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+summary: Check that `snap run --command=reload` works
+
+kill-timeout: 1m
+execute: |
+  echo "When the service snap is installed"
```
Comment (niemeyer) [inline tests/main/snap-service/task.yaml]: We've discussed standardizing both snap.yaml and task.yaml on four spaces for all indents. Not sure where we stand on that by now.

Comment (mvo5) [inline tests/main/snap-service/task.yaml]: We do, I will fix that, my editor acted up here.

File: tests/lib/snaps/test-snapd-service/meta/snap.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: test-snapd-service
+version: 1.0
+apps:
+ test-snapd-service:
+  command: bin/start
+  daemon: simple
```
Comment (niemeyer) [inline tests/lib/snaps/test-snapd-service/meta/snap.yaml]: Still just one space here. These are generally four as well (or might be, perhaps :) so they're more readable.

Merging now, though. We can fix that on the next pass.

**Pull request merged**